File: 151004/Ivanenko/DC_Lab1/DC_Lab1/DTO/StickerRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class StickerRequestTo(int? Id, string? Name) : IRequestTo
    {
        public int? Id { get; set; } = Id;

        public string? Name { get; set; } = Name;
    }
}

File: 151002/Saprin/RV/publicator/src/main/java/by/bsuir/publicator/dto/StickerRequestTo.java
package by.bsuir.publicator.dto;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigInteger;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StickerRequestTo {
    private BigInteger id;

    @Size(min = 2, max = 32)
    private String name;
    private List<BigInteger> issueIds;
}
